controller/transfer/record_transfer_manager.py

```diff
@@ -256,6 +259,12 @@ def split_record_data_and_label_data(
     return records_data, labels_data, tasks_data
 
 
+def __check_label_has_letters(label_value: str) -> bool:
+    if label_value == "" or label_value.isspace():
```

Comment:
To prevent other whitespace issues i think we should stip the value (maybe for the whole column so " test " is used as "test".

if we do it one by one then with:
```suggestion
    if label_value.strip() == "":
```

Reply:
Its a suggestion, but we would have to make sure that this is the behavior we want. I remember that we wanted to make as little changes to the uploaded data as possible. If we can ensure that the space is an error in all or most of the times then I would go with your solution.